src/mon/Monitor.cc

```diff
@@ -3057,7 +3057,7 @@ void Monitor::get_cluster_status(stringstream &ss, Formatter *f,
       const auto quorum_names = get_quorum_names();
       const auto mon_count = monmap->mon_info.size();
       ss << "    mon: " << spacing << mon_count << " daemons, quorum "
-	 << quorum_names << " (age " << quorum_age() << ")";
+	 << quorum_names << " (age " << timespan_str(now - quorum_since) << ")";
```

Comment:
Is this to automatically get the `s` unit suffix?

Reply:
Yeah (or ``m``, ``h``, ``d``, etc.)